app/models/concerns/curation_concerns/suppressible.rb

```diff
@@ -0,0 +1,31 @@
+module CurationConcerns
+  # Workflow considerations
+  module Suppressible
+    extend ActiveSupport::Concern
+
+    included do
+      # This holds the workflow state
+      property :state, predicate: Vocab::FedoraResourceStatus.objState, multiple: false
+    end
+
+    ##
+    # Used to restrict visibility on search results for a work that is inactive. If the state is not set, the
+    # default behavior is to consider the work not to be suppressed.
+    #
+    # Override this method if you have some criteria by which records should not display in the search results.
+    def suppressed?
+      return false if state.nil?
+      # This is a clumsy check only needed under RDF < 2.0 where URIs
+      # are returned as AT::Resources
```

Comment:
I believe both versions return `AT::Resources`, but the latter versions implement `==` correctly